src/frontend/app/shared/components/list/list-types/app-event/cf-app-events-config.service.ts

```diff
@@ -30,7 +30,7 @@ export class CfAppEventsConfigService extends ListConfig<EntityInfo> implements
       columnId: 'detail', headerCell: () => 'Detail', cellComponent: TableCellEventDetailComponent, cellFlex: '6'
     },
   ];
-  pageSizeOptions = [5, 25, 50];
+  pageSizeOptions = defaultPaginationPageSizeOptionsTable;
```

Comment:
We seem to be assign defaults in almost all config services, can we make pageSizeOptions optional and apply the defaults in the list component? We could do both card and table defaults this way.